src/Modules/LearningTasks/Tutor.LearningTasks.API/Public/Authoring/IActivityService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using FluentResults;
+using Tutor.LearningTasks.API.Dtos.Activities;
+
+namespace Tutor.LearningTasks.API.Public.Authoring;
+
+public interface IActivityService
+{
+    Result<ActivityDto> GetWithExamples(int id);
```

Comment:
When we change the `Example` to be part of the `Activity`, we can remove the `WithExample` suffixes in these two methods.